src/Beacon.jsx

```diff
@@ -21,11 +21,19 @@ const TOOLTIP_STATE_VARIABLES = [
   'tooltipHeight',
   'tooltipWidth',
   'tooltipHidden',
-  'tooltipAttachment',
+  'tooltipAttachmentVertical',
+  'tooltipAttachmentHorizontal',
   'appRoot',
   'appRootClassName'
 ];
 
+// Adjustments for CSS animation - $animMotionStep, $animMotionScale
+const ANIMATION_MOTION_STEP = 30;
+const ANIMATION_MOTION_SCALE = 0.8;
+
```

Comment:
I'm not entirely happy about mimicking Stylus vars this way, but there's no simple workaround before real CSS variables are usable.